### ISSUE TITLE
`${^TYPE_PARAMS_MULTISIG}` is deprecated

### ISSUE THREAD

**Issue by tobyink** [ybrliiu/p5-Sub-WrapInType#6]
https://github.com/ybrliiu/p5-Sub-WrapInType/blob/f28562501469c977c8267966745c4a21c0cf3d47/lib/Sub/WrapInType.pm#L45

`${^_TYPE_PARAMS_MULTISIG}` (with an extra underscore) is its replacement in Type::Params v2.

`${^TYPE_PARAMS_MULTISIG}` is still supported, but support will be removed in about six months.

This is because the next version of the Perl may introduce new warnings or errors related to these variables. See https://github.com/Perl/RFCs/pull/22.

This chunk of code in particular, can be written as:

```perl
use Type::Params 2.000000 qw( signature );

sub new {
  state $check = signature(
    method => 1,
    multi  => [
      {
        positional => [
          $ParamsTypes,
          $ReturnTypes,
          CodeRef,
          $Options, +{ default => sub { $DEFAULT_OPTIONS } }
        ],
      },
      {
        named_to_list => 1,
        named => [
          params  => $ParamsTypes,
          isa     => $ReturnTypes,
          code    => CodeRef,
          options => $Options, +{ default => sub { $DEFAULT_OPTIONS } }
        ],
      },
    ],
  );
  my ($class, $params_types, $return_types, $code, $options) = $check->(@_);
  
  ...;
}
```

**Comment by tobyink** [comment]
There's another usage of the variable here:

https://github.com/ybrliiu/p5-Sub-WrapInType/blob/f28562501469c977c8267966745c4a21c0cf3d47/lib/Sub/WrapInType.pm#L130

Could be written as:

```perl

sub wrap_method {
  state $check = signature(
    message => join( "\n" => (
      q{USAGE: wrap_method(\@parameter_types, $return_type, $subroutine)},
      q{    or wrap_method(params => \@params_types, isa => $return_types, code => $subroutine)},
    ) ),
    multi => [
      { positional => [ $ParamsTypes, $ReturnTypes, CodeRef ] },
      { named => [ params => $ParamsTypes, isa => $ReturnTypes, code => CodeRef ], named_to_list => 1 },
    ],
  );
  my ($params_types, $return_types, $code) = $check->(@_);

  ...;
}
```

**Comment by ybrliiu** [comment]
Thank you for your kind attention.
I'll fix it.

**Comment by ybrliiu** [comment]
I fixed it #7.
Thanks.
